includes/ajax-override.php

```diff
@@ -254,18 +256,21 @@ function tsmlfe_ajax_feedback() {
 				$chg_phone = preg_replace('/[^[:digit:]]/', '', sanitize_text_field($_POST['phone']));
 				$chg_mailing_address = stripslashes(sanitize_text_field($_POST['mailing_address']));
 				$chg_venmo = sanitize_text_field($_POST['venmo']);
-				$chg_square = sanitize_text_field($_POST['square']);
+				$chg_square = '';
+				if( isset($_POST['square']) && !empty($_POST['square']) ) {	$chg_square = sanitize_text_field($_POST['square']); }
 				$chg_paypal = sanitize_text_field($_POST['paypal']);
-				$chg_contact_1_name = sanitize_text_field($_POST['contact_1_name']);
-				$chg_contact_1_email = sanitize_text_field($_POST['contact_1_email']);
-				$chg_contact_1_phone = preg_replace('/[^[:digit:]]/', '', sanitize_text_field($_POST['contact_1_phone']));
-				$chg_contact_2_name = sanitize_text_field($_POST['contact_2_name']);
-				$chg_contact_2_email = sanitize_text_field($_POST['contact_2_email']);
-				$chg_contact_2_phone = preg_replace('/[^[:digit:]]/', '', sanitize_text_field($_POST['contact_2_phone']));
-				$chg_contact_3_name = sanitize_text_field($_POST['contact_3_name']);
-				$chg_contact_3_email = sanitize_text_field($_POST['contact_3_email']);
-				$chg_contact_3_phone = preg_replace('/[^[:digit:]]/', '', sanitize_text_field($_POST['contact_3_phone']));
-				
+				if( isset($_POST['$chg_contact_1_name']) ) {	 
```

Comment:
i think you mean `$_POST['contact_1_name']` rather than `$_POST['$chg_contact_1_name']`

Reply:
be great to use a repeat loop for these like we do with TSML

Reply:
I'll look at refactoring the code to handle contact_fields in a future update. Thx for pointing out how it's handled in TSML.